roles/confluent.variables/vars/main.yml

```diff
@@ -74,9 +74,9 @@ kafka_broker_default_group: confluent
 kafka_broker_default_log_dir: /var/log/kafka
 kafka_broker:
   server_start_file: "{{ binary_base_path }}/bin/kafka-server-start"
-  config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka/server.properties"
-  client_config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka/client.properties"
-  zookeeper_tls_client_config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka/zookeeper-tls-client.properties"
+  config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}{{ kafka_broker_config_prefix }}/server.properties"
```

Comment:
this will be problematic in a non archive install (im pretty sure) because the systemd exec start expects the properties file at `/etc/kafka/server.properties` 

At the minimum it must be documented that you can't update this unless doing archive install. or could put a logic gate in